### PR TITLE
bump api version - snippet context added

### PR DESCRIPTION
**Pull request by dimacodota**
bump api version - snippet context added
ref: https://github.com/codota/acpl-paid/pull/2071

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .gradle/
 gradle.properties
 
+bin/
```
Comment (yonip23) [inline .gitignore:4]: ?

File: src/main/java/com/tabnine/binary/requests/selection/SelectionRequest.java [hunk 2]
```diff
@@ -23,7 +23,8 @@ public class SelectionRequest {
   // text written before the suggestion start ('String name'.length)
   @SerializedName(value = "line_prefix_length")
   public Integer linePrefixLength;
-  // line_prefix_length - the part of the text that's in the suggestion ('String '.length)
+  // line_prefix_length - the part of the text that's in the suggestion ('String
+  // '.length)
```
Comment (yonip23) [inline src/main/java/com/tabnine/binary/requests/selection/SelectionRequest.java:27]: why line break here? i think you should revert this line

File: src/main/java/com/tabnine/general/StaticConfig.java [hunk 3]
```diff
@@ -18,11 +18,12 @@
 import org.jetbrains.annotations.NotNull;
 
 public class StaticConfig {
-  // Must be identical to what is written under <id>com.tabnine.TabNine</id> in plugin.xml !!!
+  // Must be identical to what is written under <id>com.tabnine.TabNine</id> in
+  // plugin.xml !!!
```
Comment (yonip23) [inline src/main/java/com/tabnine/general/StaticConfig.java:22]: also here

**Review by yonip23 — COMMENTED** [review]
please fix your format and then i'll re-review - its hard to see what changes are relevant and what not

**Review by yonip23 — APPROVED** [review]
except for my previous comments, lgtm

File: .gitignore (hunk 1 above)
Comment (dimacodota) [reply .gitignore:4]: `bin` contains compiled files (.class etc)

Screen Shot 2022-07-21 at 17 38 08

**Pull request merged**